jep/0000/README.adoc

```diff
@@ -0,0 +1,238 @@
+= JEP-0000: Jenkins Configuration as Code
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
+
+| Title
+| Jenkins Configuration as Code
+
+| Sponsor
+| https://github.com/ewelinawilkosz2[Ewelina Wilkosz] from https://github.com/praqma[Praqma]
+
+| Status
+// Uncomment the appropriate line.
+| Not Submitted :information_source:
+//| Draft :speech_balloon:
+//| Deferred :hourglass:
+//| Accepted :ok_hand:
+//| Rejected :no_entry:
+//| Withdrawn :hand:
+//| Final :lock:
+//| Replaced :dagger:
+//| Active :smile:
+
+| Type
+| Standards
+
+| Created
+| 2017-11-07
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+| JIRA
+| https://issues.jenkins-ci.org/browse/JENKINS-31094[JENKINS-31094]
+//
+//
+// Uncomment if there will be a BDFL delegate for this JEP.
+| BDFL-Delegate
+| https://github.com/ewelinawilkosz2[Ewelina Wilkosz]
```

Comment:
@ndeloof 
Is there an email on the dev list or a comment in that JIRA naming this person as BDFL delegate (and getting agreement from other contributors)? If not, would you mind if I return this to BDFL until we have that?

Reply:
sure, no worries